docs/minikube.md

```diff
@@ -0,0 +1,49 @@
+# Minikube
+
+This document details Minikube specific deployment instructions.
+Minikube is not recommended as a platform to test or develop Contour due to its network limitations.
+
+## Access your cluster
+
+On Minikube you can retrieve the external address of Contour's load balancer:
+```
+$ minikube service -n heptio-contour contour --url
+http://192.168.99.100:30588
+```
+
+## Configuring DNS
+
+On Minikube, you can fake DNS by editing `/etc/hosts` or you can use the provided example and not have to modify dns on your local machine.
+
+## Deploy KUARD example application.
+
+```
+$ kubectl apply -f https://j.hept.io/contour-kuard-minikube-example
+```
+
+This example yaml specifies `kuard.192.168.99.100.nip.io` as a specific ingress backend for kuard.
+It uses nip.io and the minikube ip address to have kuard only respond to http://kuard.192.168.99.100.nip.io.
```

Comment:
"It uses nip.io and the Minikube IP address so that kuard responds only at http://kuard.192.168.99.100.nip.io."